__init__.py

```diff
@@ -11,72 +11,90 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-import datetime
+"""Skill to display a home screen (a.k.a. idle screen) on a GUI enabled device."""
 import json
-import os
-import time
-import requests
 from datetime import datetime, timedelta
-from os import path
 from pathlib import Path
+from typing import Optional
 
 from mycroft.messagebus.message import Message
-from mycroft.skills import MycroftSkill, resting_screen_handler, intent_handler, intent_file_handler
-from mycroft.skills.skill_loader import load_skill_module
-from mycroft.skills.skill_manager import SkillManager
+from mycroft.skills import MycroftSkill, resting_screen_handler, intent_file_handler
```

Comment:
Personally I've been moving all intents to use the `intent_handler` decorator since it handles both Adapt and Padatious. But I can also see the argument that the "file" version highlights that this only uses Padatious.

Reply:
This took me aback when I saw it too... until I realized what it was.  I will change it.